Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM openjdk:9-jdk-slim AS builder
+WORKDIR /usr/src/app
+COPY . /usr/src/app
+RUN ./gradlew clean build
+
+FROM openjdk:9-jre-slim
```

Comment:
base docker image seems to be written twice

Reply:
I'm leveraging [multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build) to keep things compact and cache gradle specific scripts, libs, etc.